src/FetchAuthTokenInterface.php

```diff
@@ -40,4 +40,17 @@ public function fetchAuthToken(callable $httpHandler = null);
   * @return string a key that may be used to cache the auth token.
   */
   public function getCacheKey();
+
+  /**
+   * Returns an associative array with the token and expiration time
```

Comment:
Might want to make clear that it's the expiration time in seconds since the Unix epoch.